services/dask-sidecar/Dockerfile

```diff
@@ -0,0 +1,152 @@
+ARG PYTHON_VERSION="3.8.10"
+FROM python:${PYTHON_VERSION}-slim-buster as base
+#
+#  USAGE:
+#     cd sercices/dask-sidecar
+#     docker build -f Dockerfile -t dask-sidecar:prod --target production ../../
+#     docker run dask-sidecar:prod
+#
+#  REQUIRED: context expected at ``osparc-simcore/`` folder because we need access to osparc-simcore/packages
+
+LABEL maintainer=pcrespov
+
+RUN set -eux \
+  && apt-get update \
+  && apt-get install -y --no-install-recommends \
+  iputils-ping \
+  curl \
+  gosu \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* \
+  # verify that the binary works
+  && gosu nobody true
+
+
+# simcore-user uid=8004(scu) gid=8004(scu) groups=8004(scu)
+ENV SC_USER_ID=8004 \
+  SC_USER_NAME=scu \
+  SC_BUILD_TARGET=base \
+  SC_BOOT_MODE=default
+
+RUN adduser \
+  --uid ${SC_USER_ID} \
+  --disabled-password \
+  --gecos "" \
+  --shell /bin/sh \
+  --home /home/${SC_USER_NAME} \
+  ${SC_USER_NAME}
+
+
+ENV LANG=C.UTF-8 \
+  PYTHONDONTWRITEBYTECODE=1 \
+  VIRTUAL_ENV=/home/scu/.venv
+
+ENV PATH="${VIRTUAL_ENV}/bin:$PATH"
+
+ENV SWARM_STACK_NAME="" \
+  SIDECAR_INPUT_FOLDER=/home/scu/input \
+  SIDECAR_OUTPUT_FOLDER=/home/scu/output \
+  SIDECAR_LOG_FOLDER=/home/scu/log
+
+
+#
+# HEALTHCHECK --interval=60s \
+#   --timeout=60s \
+#   --start-period=30s \
+#   --retries=3 \
+#   CMD ["curl", "-Lf", "http://scheduler:8787/health"]
```

Comment:
hmm no healthcheck?

Reply:
fixed. I moved all healthchecks to Dockerfile